SUPPORTING_NEW_SERVICES.md

```diff
@@ -0,0 +1,108 @@
+
+Notes/reminders:
+* Clearly document which APIs must be enabled in the Developers Console's API Manager.
+* Versioning in documentation is automatically updated by the script `utilities/update_docs_version.sh`.  Be sure to examine that script to make sure any version-dependent documentation will be updated properly upon release.
+
+### Workflow
+
+New services should be created in a branch based on `master`.  The branch name should include the suffix "-alpha".  For example, while developing `gcloud-java-pubsub`, all Pub/Sub related work should be done in `pubsub-alpha`.  All code should be submitted through pull requests from a branch on a forked repository.  Limiting pull request size is very helpful for reviewers.  All code that is merged into the branch should be standalone and well-tested.  Any todo comments in the code should have an associated Github issue number for tracking purposes.  You should periodically pull updates from the master branch, especially if there are project-wide updates or if relevant changes have been made to the core utilities library, `gcloud-java-core`.
+
+Create at least two milestones (stable and future) for your service and an issue tag with the service name.  Create issues for any to-do items and tag them appropriately.  This keeps an up-to-date short-term to-do list and also allows for longer term roadmaps.
+
+Be sure you've configured the base folder's `pom.xml` correctly.
+  * Add your module to the base directory's `pom.xml` file under the list of modules (in alphabetical order).
+  * Add your module to the javadoc packaging settings.  See PR #802 for an example.
+  [comment]: # (TODO: uncomment when gcs-nio branch is merged into master) * Add your example to the assembler plugin.  PR #839 includes examples of using appassembler.
+
+When your service is complete, contact the service owners to get a review.  The primary purpose of this review is to make sure that the gcloud-java service interacts with the Cloud service properly.  Present the reviewers with a link to the Github repository, as well as your (updated) design document that details the API.
+
+### Closing remarks
+
+* Efforts should be made to maintain the current style of the repository and a consistent style between gcloud-java services.
+  * We anticipate that people will sometimes use multiple `gcloud-java` services, so we don't want differences in conventions from library to library.  Look at existing `gcloud-java` services to see coding and naming conventions.
+  * Codacy is configured to report on pull requests about style issues.  Whenever possible, those comments should be addressed.  Coders and reviewers should also run a linter on pull requests, because the Codacy tool may not catch all style errors.
+* When weighing which services to add, consider that a hand-crafted `gcloud-java` service API is especially useful if it can abstract away and/or make java-idiomatic significant parts of a service's autogenerated API.
```

Comment:
> it can abstract away and/or make java-idiomatic significant parts of a service's autogenerated API.

Instead I would say "if it can simplify the usability of the auto-generated client"